chrony/service/running.sls

```diff
@@ -15,3 +15,5 @@ chrony-service-running-service-running:
     - enable: True
     - require:
       - sls: {{ sls_config_file }}
+    - watch:
+      - file: chrony-config-file-file-managed
```

Comment:
@sschols Thanks for providing this PR.  This bug is actually captured here but no-one got around to fixing this formula yet, so thank you for doing that:

* https://tree.taiga.io/project/myii-saltstack-formulas/us/176

In terms of the fix itself, this is actually all that is required:

```suggestion
    - watch:
      - sls: {{ sls_config_file }}
```

I.e. The `require` needs to be changed to `watch` -- that's all.  Perhaps easier to see here:

* https://github.com/saltstack-formulas/template-formula/pull/193/files